src/config/config-manager.js

```diff
@@ -0,0 +1,18 @@
+
+export default function ConfigManager () {
+  this.seenConfigs = []
+}
+
+export function qualifiedConfig (config) {
```

Comment:
Let's assign a numeric value to the 'goodness' of particular configuration. Than we can just compare which one is better on startup